pyproject.toml

```diff
@@ -14,6 +14,7 @@ azure-core = "<2.0.0"
 msrest = "<1.0.0"
 azure-mgmt-core = "<2.0.0"
 aiohttp = "^3.8.4"
+langsmith = "0.1.48"
```

Comment:
We should probably also be more permissive in the Langsmith versions we support, or else our users may get conflicts when they install our SDK if they already using Langsmith.